pkg/daemon/rpm-ostree.go

```diff
@@ -292,12 +237,6 @@ func (r *RpmOstreeClient) PullAndRebase(container string, keep bool) (imgid stri
 		return
 	}
 
-	// By default, delete the image.
-	if !keep {
-		// Related: https://github.com/containers/libpod/issues/2234
-		exec.Command("podman", "rmi", imgid).Run()
-	}
```

Comment:
are we leaving `osImageContentDir` around? is there any safe place we could get rid of it? (just thinking out loud)

Reply:
osImageContentDir is keeping machine-os-content extracted using `oc extract image` in  a temporary dir /run/mco-machine-os-content/ . This will go away once system reboots after update.

Reply:
Nice! I missed that, thanks Sinnny :)

Reply:
np :)